docs/get-started/sveltekit.md

```diff
@@ -0,0 +1,350 @@
+
+##### `navigation.replaceState`
+
+Type: TK
+
+A callback that will be called whenever `replaceState` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.invalidate`
+
+Type: TK
+
+A callback that will be called whenever `invalidate` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.invalidateAll`
+
+Type: TK
+
+A callback that will be called whenever `invalidateAll` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.afterNavigate`
+
+Type: TK
+
+An object that will be passed to the `afterNavigate` function, which will be invoked when the `onMount` event fires.
+
+#### `stores`
+
+Type: `{ navigating?: {}, page?: {}, updated?: boolean }`
+
+Provides mocks for the `$app/stores` module.
+
+##### `stores.navigating`
+
+Type: TK
+
+A partial version of the `navigating` store.
+
+##### `stores.page`
+
+Type: TK
+
+A partial version of the `page` store.
+
+##### `stores.updated`
+
+Type: boolean
+
+A boolean representing the value of `updated` (you can also access `stores.check()` which will be a noop).
+
+### Options
+
+You can pass an options object for additional configuration if needed:
+
+```js
+// .storybook/main.js
+import * as path from 'path';
+
+export default {
+  // ...
+  framework: {
+    name: '@storybook/sveltekit',
+    options: {
+      // ...
+    },
+  },
+};
+```
+
+The available options are:
+
+#### `builder`
+
+Type: `Record<string, any>`
+
+Configure options for the [framework's builder](../api/main-config-framework.md#optionsbuilder). For Sveltekit, available options can be found in the [Vite builder docs](../builders/vite.md).
+
+## Troubleshooting
+
+### Error when starting Storybook
+
+When starting Storybook after upgrading to v7.0, it may quit with the following error:
+
+```sh
+ERR! SyntaxError: Identifier '__esbuild_register_import_meta_url__' has already been declared
+```
+
+This can occur when manually upgrading from 6.5 to 7.0. To resolve it, you'll need to remove the `svelteOptions` property in `.storybook/main.js`, as that is not supported (and no longer necessary) in Storybook 7+ with SvelteKit.
+
+### Error in preview canvas
+
+When running Storybook, some stories may not load. Instead they show the following error in the preview canvas:
+
+```sh
+Cannot read properties of undefined (reading 'disable_scroll_handling')
+```
+
+You'll experience this if anything in your story is importing from `$app/forms` or `$app/navigation`, which is currently not supported. To get around this, separate your component into a shallow parent component that imports what's needed and passes it to a child component via props. This way you can write stories for your child component and mock any of the necessary modules by passing props in.
+
```

Comment:
This error is no longer relevant since https://github.com/storybookjs/storybook/pull/25138, 8.0.0-alpha.2 . Was there a deliberate reason to include this?

```suggestion
```